terraform/gcp/modules/monitoring/variables.tf

```diff
@@ -77,6 +82,11 @@ variable "prober_fulcio_url" {
   default = "http://fulcio-server.fulcio-system.svc"
 }
 
+variable "prober_timestamp_url" {
+  type    = string
+  default = "http://timestamp-server.fulcio-system.svc"
```

Comment:
```suggestion
  default = "http://timestamp-server.timestamp-system.svc"
```

Reply:
:+1: I'll use "tsa-system" since that's what is used elsewhere

Reply:
hmm actually there is some "timestamp-system" as well: will standardize